src/github.ts

```diff
@@ -190,4 +195,84 @@ export class GitHub {
       url: artifactURL
     };
   }
+
+  public static async writeBuildSummary(opts: BuildSummaryOpts): Promise<void> {
+    // can't use original core.summary.addLink due to the need to make
+    // EOL optional
+    const addLink = function (text: string, url: string, addEOL = false): string {
+      return `<a href="${url}">${text}</a>` + (addEOL ? os.EOL : '');
+    };
+
+    const refsSize = Object.keys(opts.exportRes.refs).length;
+
+    // prettier-ignore
+    const sum = core.summary
+      .addHeading('Docker Build summary', 1)
+      .addRaw(`<p>`)
+        .addRaw(`For a detailed look at the build, download the following build record archive and import it into Docker Desktop's Builds view. `)
+        .addBreak()
+        .addRaw(`The build file provides insights into your build, such as timing, dependencies, results, logs, traces, and more. `)
```

Comment:
Re-reading this, and thinking that "a file provides insights" can be improved, suggestion:

```suggestion
        .addRaw(`Build records include details such as timing, dependencies, results, logs, traces, and other information about a build. `)
```

Reply:
https://github.com/docker/actions-toolkit/actions/runs/9095301220?pr=324#summary-24998248180

![image](https://github.com/docker/actions-toolkit/assets/1951866/3f901f02-9876-45ca-9809-ee57c605a0dd)

Reply:
LGTM